### PR TITLE
Some more maintenance for the arcana mod

### PR DESCRIPTION
**Pull request by Cyrano7**

### CODE REVIEW

File: data/mods/Arcana/tool_armor.json [hunk 1]
```diff
@@ -266,23 +254,15 @@
...
+            "effects" : [ { "id" : "cleric_warding", "duration" : 250 } ]
         },
         "artifact_data" : {
           "effects_worn": ["AEP_PSYSHIELD", "AEP_STR_DOWN"]
         },
         "color" : "white",
-        "encumbrance" : 0,
         "price" : 30000,
         "material" : ["silver", "essencemat"],
-        "coverage" : 0,
```
Comment (illi-kun) [inline data/mods/Arcana/tool_armor.json]: I see no default value for ```coverage``` defined, so removing of this line causes an error. Please run Cataclysm as  ```./cataclysm --check-mods``` (if you are on Linux) for testing your changes.

File: data/mods/Arcana/tool_armor.json [hunk 2]
```diff
@@ -256,8 +246,6 @@
         "id" : "cleric_ring",
         "name" : "ring of the fallen angel",
         "name_plural" : "rings of the fallen angel",
-        "volume" : 0,
-        "material_thickness" : 0,
```
Comment (illi-kun) [inline data/mods/Arcana/tool_armor.json]: There is no default value for ```material_thickness``` defined.

**Comment by Cyrano7** [comment]
@illi-kun itype.h struct islot_armor

**Comment by Cyrano7** [comment]
Am I not looking at the right part.

**Comment by illi-kun** [comment]
Take a look on ```item_factory.cpp```:
```
void Item_factory::load( islot_armor &slot, JsonObject &jo )
{
    slot.encumber = jo.get_int( "encumbrance", 0 );
    slot.coverage = jo.get_int( "coverage" );
    slot.thickness = jo.get_int( "material_thickness" );
    slot.env_resist = jo.get_int( "environmental_protection", 0 );
    slot.warmth = jo.get_int( "warmth", 0 );
    slot.storage = jo.get_int( "storage", 0 );
    slot.power_armor = jo.get_bool( "power_armor", false );
    slot.covers = jo.has_member( "covers" ) ? flags_from_json( jo, "covers", "bodyparts" ) : 0;
    ...
```

**Comment by Cyrano7** [comment]
ahh. Hold up let me add the defaults in another PR real quick.

**Comment by illi-kun** [comment]
OK, but please do it as small as possible, changing of two lines will be ideal.

**Pull request merged**